puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-plugins-pull-request.yaml

```
@@ -37,7 +38,7 @@
             - fast
     execution-strategy:
       combination-filter: >
-        ((database == 'postgresql') && ((ruby == '2.5') || (ruby == '2.6'))) || ((database == 'sqlite3') && (ruby == '2.5'))
+        ((database == 'postgresql') && ((ruby == '2.5') || (ruby == '2.6') || (ruby == '2.7')) || ((database == 'sqlite3') && (ruby == '2.5'))
```

In 03c7f4227ebc4509149122160d9fccdb71ab790a we dropped sqlite3 testing for core - should we also drop it for plugins?

If not, we can still simplify this to:

```suggestion
        (database == 'postgresql') || ((database == 'sqlite3') && (ruby == '2.5'))
```

cc @tbrisker

---

Sure, but please in a separate PR.

---

Note my suggestion still keeps sqlite, just drops a redundant comparison for PG

---

Ah, yes. My answer was to "should we also drop it for plugins?".

---

> Note my suggestion still keeps sqlite, just drops a redundant comparison for PG

there we go...